tests/integration/cloudformation/test_template_engine.py

```diff
@@ -246,6 +246,14 @@ def test_get_azs_function(self, deploy_cfn_template):
         zone = "us-east-1a"  # TODO parametrize
         assert zone in deployed.outputs["Zones"]
 
+    def test_sub_not_ready(self, deploy_cfn_template):
```

Comment:
```suggestion
    @markers.parity.aws_validated
    def test_sub_not_ready(self, deploy_cfn_template):
```

Reply:
Thanks, nice use for the new marker! I've integrated this into a new commit